### DISCUSSION TITLE
Prototype / design history

### DISCUSSION THREAD

**Discussion by michaelcattell** [dwp/design-system-community-backlog#77] [Q&A]
A discussion happened on slack this morning (5th November) around how we track design decisions within the prototype, comments added with uploads of images.
Original author asked:
What does yours look like? Please pop on a screenshot.
Which service are you?
What do you think is good about yours / could be better?

**Comment by michaelcattell** [comment]
(Allan Robinson)
![image](https://user-images.githubusercontent.com/52162960/140498320-76bce94c-7765-4a37-a4fb-aa4f8dac97da.png)

**Reply by michaelcattell** [reply]
![image](https://user-images.githubusercontent.com/52162960/140498701-8d67c0b6-b7ce-4148-b989-e91a18c3b474.png)
![image](https://user-images.githubusercontent.com/52162960/140498709-6b94eacb-8686-4bd6-bbc2-d1b8b52e5dcf.png)

**Comment by michaelcattell** [comment]
K Watts
![image](https://user-images.githubusercontent.com/52162960/140498563-20685259-5a27-444d-a8e2-b6d832d7a8f7.png)
![image](https://user-images.githubusercontent.com/52162960/140498578-01610251-bc66-481f-adab-7370ea22fba8.png)
![image](https://user-images.githubusercontent.com/52162960/140498587-4263193b-37f3-4c83-b0ff-9247334bd14b.png)

**Comment by michaelcattell** [comment]
N Holroyd
![image](https://user-images.githubusercontent.com/52162960/140498653-08384ffe-9b6e-4b3c-a897-eb6729494f3c.png)

**Comment by michaelcattell** [comment]
P Jobes
![image](https://user-images.githubusercontent.com/52162960/140498755-d3d1cbfd-5cba-426d-b725-ecb981db4a8e.png)

**Comment by michaelcattell** [comment]
R Derbyshire
![image](https://user-images.githubusercontent.com/52162960/140498796-95ff1a36-7211-4d35-9144-966422992ce4.png)

**Comment by michaelcattell** [comment]
G Joyce
I find sticking to sprints and adding dates useful to see how things change over time based on learnings.
There is a high level index which usually links to a specific sprint index page which has more context about the specific thing we’re doing that sprint with any related hypothesis. User research is captured raw in Mural then analysed and synthesised into confluence and ppt decks for playbacks and show and tells.
Some of these link out to other prototype URLs because there was some custom patterns and data in the other prototype that was easy to just use quickly rather than duplicate all the styles and logic
![image](https://user-images.githubusercontent.com/52162960/140498876-79cf9eff-f9c2-4017-a776-5cc081cf5e7a.png)
![image](https://user-images.githubusercontent.com/52162960/140498887-6fb8b676-2791-4a4b-a82b-c8a90744459b.png)

**Reply by michaelcattell** [reply]
**There is also this done by DfE which is great.**
https://design-history.herokuapp.com/
https://bat-design-history.netlify.app/

**Comment by michaelcattell** [comment]
B Fraser / M Wake:
![image](https://user-images.githubusercontent.com/52162960/140513142-596c8155-ec8a-4ce0-b8fb-7c172c4dc46d.png)

**Comment by michaelcattell** [comment]
![image](https://user-images.githubusercontent.com/52162960/140513378-490ad680-4458-4f00-8536-37637b6b5902.png)

**Comment by michaelcattell** [comment]
![image](https://user-images.githubusercontent.com/52162960/140513410-c44e6b31-ff96-4d00-86d4-ff4e9ad64f85.png)

**Comment by steve-oconnor** [comment]
A lot of these are very much like a mini style guide / design system of their own! Fascinating to see. In DfE's Services Delivery Division it's mainly design histories: https://sdd-design-history.netlify.app/

**Comment by Al-Robinson** [comment]
<img width="773" alt="Screenshot 2021-12-02 at 15 20 29" src="https://user-images.githubusercontent.com/77453029/144451711-bf299fb8-26db-4103-babd-1f8fc2d90716.png">
<img width="762" alt="Screenshot 2021-12-02 at 15 20 40" src="https://user-images.githubusercontent.com/77453029/144451716-9206357c-3059-4cda-8d6a-7bb9b450064a.png">
<img width="755" alt="Screenshot 2021-12-02 at 15 20 46" src="https://user-images.githubusercontent.com/77453029/144451717-db6dae03-1c62-4ed5-b792-4cf7783717f3.png">
<img width="755" alt="Screenshot 2021-12-02 at 15 20 55" src="https://user-images.githubusercontent.com/77453029/144451720-0b8f10d1-e487-4f3c-b6d3-61ebc4c4abff.png">

This is our resolved design in winter fuels
